.github/workflows/iroha2-dev-pr-wasm.yaml

```diff
@@ -20,31 +20,35 @@ concurrency:
   cancel-in-progress: true
 
 env:
-  RUSTC_BOOTSTRAP: 1
+  RUSTUP_TOOLCHAIN: nightly-2022-08-15
```

Comment:
Careful. Now line 38 is using the `nightly` lints. This is a non-trivial change.

Reply:
Indeed, the changes to `Lints` and `no_std compatibility` steps are not refactor because they switches to nightly channel